sensor/handler.go

```diff
@@ -1,23 +1,4 @@
-// Copyright (C) 2021 Greenbone Networks GmbH
-//
-// SPDX-License-Identifier: AGPL-3.0-or-later
-//
-// This program is free software: you can redistribute it and/or modify
-// it under the terms of the GNU Affero General Public License as
-// published by the Free Software Foundation, either version 3 of the
-// License, or (at your option) any later version.
-//
-// This program is distributed in the hope that it will be useful,
-// but WITHOUT ANY WARRANTY; without even the implied warranty of
-// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-// GNU Affero General Public License for more details.
-//
-// You should have received a copy of the GNU Affero General Public License
-// along with this program.  If not, see <http://www.gnu.org/licenses/>.
-
-// Package handler contains various message handler for sensors and initializes
-// MQTT connection
-package handler
+package sensor
```

Comment:
License header got removed here.
It seems some other files not touched by this PR are also missing the license header. I can make a small separate PR for all of that later.

Reply:
That would be nice.